service/lib/ops/policyOps.js

```diff
@@ -180,16 +180,26 @@ const policyOps = {
   createPolicy: function createPolicy (params, cb) {
     const { id, version, name, organizationId, statements } = params
 
-    insertPolicies(db, [{
-      id: id,
-      version: version,
-      name: name,
-      org_id: organizationId,
-      statements: statements
-    }], (err, result) => {
-      if (err) return cb(err)
-
-      policyOps.readPolicy({ id: result.rows[0].id, organizationId }, cb)
+    const checkIdSql = SQL`
+      SELECT id
+      FROM policies
+      WHERE id = ${id}
+    `
+    db.query(checkIdSql, (err, res) => {
```

Comment:
@paolochiodi I went with the first one. I usually don't like to rely on external error messages but it allowed me to wrap the check in a function and have better readability. Thanks for the suggestion!